src/Redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    items: [],
+    status: 'Under construction',
+  },
+  reducers: {
+    checkstate: (state) => {
+      state.status = 'Under construction';
+    },
+  },
+});
+
```

Comment:
- [ ] Please consider exporting the action creator `checkstatus` of the `categoriesSlice` the same way in the previous comment like `const { addPotato, addTomato } = vegetablesSlice.actions;`